tests.py

```diff
@@ -337,6 +337,25 @@ def test_as_var(self):
         self.static.assert_called_with('static.png')
 
 
+class DjangoNowTest(SimpleTestCase):
+    def setUp(self):
+        self.env = Environment(extensions=[DjangoNow])
+    
+    def test_simple(self):
+        template = self.env.from_string("{% now 'Y' %}")
+        expected = timezone.now().strftime('%Y')
```

Comment:
@MoritzS I realize `timezone.now()` always returns a datetime object using the UTC timezone, while the `now` tag returns a datetime using the current timezone. So it would be better to use `timezone.localtime(timezone.now())` instead. I will try to look into the possibility of mocking during tests.